tpm2/src/getkey/main.c

```diff
@@ -229,11 +275,18 @@ int main(int argc, char **argv)
     BIO_free_all(bio);
     SSL_CTX_free(ctx);
     kmyth_clear_and_free(clientPrivateKey_data, clientPrivateKey_size);
+    free(sdo_orig_fn);
+    if (authString != NULL)
+    {
+       kmyth_clear(authString); 
+    }
+    kmyth_clear(ownerAuthPasswd);
     return 1;
   }
 
   // Done with unsealed key buffer, so clear and free this memory
   kmyth_clear_and_free(clientPrivateKey_data, clientPrivateKey_size);
+  free(sdo_orig_fn);
```

Comment:
If we do move the free(sdo_orig_fn) call earlier, this would not be needed at this point.